polymorphic/query.py

```diff
@@ -132,11 +132,11 @@ def as_manager(cls):
     as_manager.queryset_only = True
     as_manager = classmethod(as_manager)
 
-    def bulk_create(self, objs, batch_size=None):
+    def bulk_create(self, objs, **kwargs):
```

Comment:
This looks fine. I would probably err towards copying the full method signature rather than allowing kwargs  https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/django/db/models/query.py#L463

Reply:
I agree. I will update based on your suggestion. The more I thought about it, the more I think what I did violates the 'explicit is better than implicit' principle. Thanks for the review!